pkg/reconciler/v1alpha1/configuration/resources/revision.go

```diff
@@ -38,11 +39,15 @@ func MakeRevision(config *v1alpha1.Configuration, buildRef *corev1.ObjectReferen
 
 	UpdateRevisionLabels(rev, config)
 
-	// Populate the Configuration Generation annotation.
 	if rev.Annotations == nil {
 		rev.Annotations = make(map[string]string)
 	}
 
+	// Set KPA annotation as default if it is empty
```

Comment:
As discussed: probably better in `revision_defaults`.

Reply:
Reverted the changes in `revision.go`. Will set the annotation in `revision_defaults.go` in another PR.